jbpm-designer-backend/src/test/java/org/jbpm/designer/bpmn2/utils/Bpmn2Loader.java

```diff
@@ -24,13 +25,28 @@
     // It is by design (Unmarshaller = marshaller and vice versa)
     IDiagramProfile.IDiagramUnmarshaller marshaller = profile.createUnmarshaller();
 
+    // list of base workitem names
+    private static final List<String> baseWorkItemNames = new ArrayList<String>(Arrays.asList("Email",
+                                                                         "HelloWorkItemHandler",
+                                                                         "Log",
+                                                                         "Rest",
+                                                                         "WebService"));
+
     public Bpmn2Loader(Class testClass) {
         this.testClass = testClass;
     }
 
     public Definitions loadProcessFromJson(String fileName,
                                            String zOrderEnabled,
-                                           String bpsimDisplay) throws Exception {
+                                           String bpsimDisplay,
+                                           List<String> workitemNames) throws Exception {
+
+        List<String> workitemNamesList = new ArrayList<>();
```

Comment:
I think there is no need for this variable:
```java
if (workItemNames == null) {
    workItemNames = new ArrayList<> (baseWorkItemNames);
} else {
    workItemNames.addAll(baseWorkItemNames);
}
```

Please notice that `workitemNames` should be by java conventions `workItemNames`. Sorry that I didn't hint you this in first round of review.